alohomora/req.py

```diff
@@ -42,26 +42,26 @@
 try:
     from u2flib_host import u2f, exc
     from u2flib_host.constants import APDU_USE_NOT_SATISFIED, APDU_WRONG_DATA
+
     U2F_SUPPORT = True
 except ImportError:
     pass
 
 try:
-    input = raw_input #pylint: disable=redefined-builtin,invalid-name
+    input = raw_input  # pylint: disable=redefined-builtin,invalid-name
 except NameError:
     pass
 
-
 LOG = logging.getLogger('alohomora.req')
-
+csrf="";
```

Comment:
DUO looks for the csrf token when requesting for TFA. So made that as global var.